### ISSUE TITLE
Documentation has misleading aria-expanded attribute

### ISSUE THREAD

**Issue by downFast** [twbs/bootstrap#23903]
Hi all
currently the documentation says:

```
 <li class="nav-item">
    <a class="nav-link active" id="pills-home-tab" data-toggle="pill" href="#pills-home" role="tab" aria-controls="pills-home" aria-expanded="true">Home</a>
  </li>
  <li class="nav-item">
    <a class="nav-link" id="pills-profile-tab" data-toggle="pill" href="#pills-profile" role="tab" aria-controls="pills-profile" aria-expanded="true">Profile</a>
  </li>
```

It has twice`aria-expanded="true"` while one of them should be set to `false`. Also on the section **_And with vertical pills._** we have `aria-expanded="true"` for all of them. Yet in the documentation it says  _**If the collapsible element is closed by default, the attribute on the control element should have a value of aria-expanded="false".**_

Therefore either I am not getting it right, or there is a mistake in the docs.

**Comment by patrickhlauke** [comment]
Interestingly, the whole tabs implementation is, on closer inspection, borked. Working on a fundamental fix at the moment, but in the meantime disregard the incorrect advice there for now...

**Comment by patrickhlauke** [comment]
See https://github.com/twbs/bootstrap/pull/23907

**Comment by downFast** [comment]
maybe it'd also be a good idea to mention "**_tabs_**" too in the description as it is only mentioning **_collapse_** here https://getbootstrap.com/docs/4.0/components/collapse/#accessibility
